pyfolio/tests/test_timeseries.py

```diff
@@ -289,7 +289,7 @@ def test_sharpe_2(self, returns, rolling_sharpe_window, expected):
             returns, rolling_sharpe_window).values.tolist()), expected)
 
     @parameterized.expand([
-        (simple_rets, 0.010766923838471554)
+        (simple_rets, 0.10376378866671222)
```

Comment:
https://github.com/quantopian/pyfolio/commit/bb13b0a5c6166a230879d941389886aeed78bbe0. You're right that it should be a BUG commit as the original implementation wasn't correct.